frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -35,6 +35,8 @@ export type OverviewItem = {
   configurations?: K8sResourceKind[];
   ksservices?: K8sResourceKind[];
   revisions?: K8sResourceKind[];
+  pipeline?: K8sResourceKind[];
```

Comment:
This should be `K8sResourceKind`
```suggestion
  pipeline?: K8sResourceKind;
```